### PR TITLE
fix changed instance name and add auto zone selection

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- instance name changed to eval-instance (was eval-ZONE)
- let Apigee automatically pick a ZONE for the instance for TRIAL orgs

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by yuriylesyuk — APPROVED** [review]

**Review by yuriylesyuk — APPROVED** [review]

File: tools/apigee-x-trial-provision/apigee-x-trial-provision.sh [hunk 1]
```diff
@@ -199,18 +199,17 @@ if [[ "$NETWORK" =~ ^projects/.*/networks.*$ ]]; then
...
+      --purpose=VPC_PEERING --project="$PROJECT" || echo "Failed to create - ingoring assuming it already exists"
   else
-    CIDR_ADDRESS=$(echo "$PEERING_CIDR" | cut -d/ -f1)
-    CIDR_MASK=$(echo "$PEERING_CIDR" | cut -d/ -f2)
-    PEERING_CIDR_FLAGS="--prefix-length=$CIDR_MASK --addresses=$CIDR_ADDRESS"
+    RANGE_START="$(echo "$PEERING_CIDR" | cut -d/ -f1)"
+    RANGE_PREFIX_LENGTH="$(echo "$PEERING_CIDR" | cut -d/ -f2)"
+    gcloud compute addresses create google-managed-services-default --global  --addresses="$RANGE_START" --prefix-length="$RANGE_PREFIX_LENGTH" \
+      --description="Peering range for Google Apigee X Tenant" --network="$NETWORK" \
+      --purpose=VPC_PEERING --project="$PROJECT" || echo "Failed to create - ingoring assuming it already exists"
```
Comment (laughingbiscuit) [inline tools/apigee-x-trial-provision/apigee-x-trial-provision.sh:210]: *ignoring

File: tools/apigee-x-trial-provision/apigee-x-trial-provision.sh [hunk 2]
```diff
@@ -199,18 +199,17 @@ if [[ "$NETWORK" =~ ^projects/.*/networks.*$ ]]; then
 else
   echo "Step 4.1: Define a range of reserved IP addresses for your network. "
   if [ -z "$PEERING_CIDR" ]; then
-    PEERING_CIDR_FLAGS="--prefix-length=23"
+    gcloud compute addresses create google-managed-services-default --global --prefix-length=23 \
+      --description="Peering range for Google Apigee X Tenant" --network="$NETWORK" \
+      --purpose=VPC_PEERING --project="$PROJECT" || echo "Failed to create - ingoring assuming it already exists"
```
Comment (laughingbiscuit) [inline tools/apigee-x-trial-provision/apigee-x-trial-provision.sh:204]: *ignoring

**Review by laughingbiscuit — APPROVED** [review]
typos, but lgtm - thanks

**Pull request merged**